src/Core/Framework/Increment/AbstractIncrementer.php

```diff
@@ -27,6 +27,15 @@ abstract public function list(string $cluster, int $limit = 5, int $offset = 0):
 
     abstract public function reset(string $cluster, ?string $key = null): void;
 
+    /**
+     * @deprecated tag:v6.8.0 - reason:visibility-change - Will become abstract
+     *
+     * @param array<string> $keys
+     */
+    public function delete(string $cluster, array $keys = []): void
+    {
```

Comment:
should we triggerThrow a deprecation in this method? with this, devs should be notified about that and implement that method